Programm started at: 2025-10-13 02:03:28

Running in version 1.2[2025-10-13 23:33:02]
Git repositories synchronized
	

	NoneType: None


[2025-10-13 23:33:02]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-10-14 02:03:22

Running in version 1.2

[2025-10-14 13:36:38]
Error 1 at reading or storing sensor data.
	I2C read failed

	Traceback (most recent call last):
  File "/home/raspi/CO2_Project/main.py", line 211, in <module>
    g_co2_sensor.read_all()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 192, in read_all
    self.read_logorithm_results()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 188, in read_logorithm_results
    raise Exception (pigpio.error_text(b))
Exception: I2C read failed


[2025-10-14 23:32:56]
Git repositories synchronized
	

	NoneType: None


[2025-10-14 23:32:56]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-10-15 02:02:46

Running in version 1.2

[2025-10-15 23:32:20]
Git repositories synchronized
	

	NoneType: None


[2025-10-15 23:32:20]
Terminating programm
	

	NoneType: None


New initialisation demanded at: 2025-10-16 02:03:37

Running in version 1.2

[2025-10-16 21:50:04]
Error 1 at reading or storing sensor data.
	'I2C read failed'

	Traceback (most recent call last):
  File "/home/raspi/CO2_Project/main.py", line 211, in <module>
    g_co2_sensor.read_all()
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 191, in read_all
    if self.data_available():
       ^^^^^^^^^^^^^^^^^^^^^
  File "/home/raspi/CO2_Project/ccs811LIBRARY.py", line 172, in data_available
    value = self.pi.i2c_read_byte_data(self.device, CSS811_STATUS)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/pigpio.py", line 2874, in i2c_read_byte_data
    return _u2i(_pigpio_command(self.sl, _PI_CMD_I2CRB, handle, reg))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/pigpio.py", line 1011, in _u2i
    raise error(error_text(v))
pigpio.error: 'I2C read failed'


